Reconstruction/main.cxx

```diff
@@ -369,8 +370,14 @@ bool AreVectorsOrthogonal()
   double YZ = vtkMath::Dot(Y, Z);
   double ZX = vtkMath::Dot(Z, X);
 
-  if (XY == 0 && YZ == 0 && ZX == 0)
-    return true;
+  double epsilon = 10e-6;
+  if (vtkMathUtilities::FuzzyCompare(XY,0.0,epsilon) &&
+     vtkMathUtilities::FuzzyCompare(YZ,0.0,epsilon) &&
+     vtkMathUtilities::FuzzyCompare(ZX,0.0,epsilon))
```

Comment:
In VTK style we add spaces after the comma. Also I prefer "&&" at the beginning of the line.